lifetracker-api/lifetracker-schema.sql

```diff
@@ -0,0 +1,43 @@
+CREATE TABLE users (
+    id SERIAL PRIMARY KEY,
+    email VARCHAR(255) NOT NULL UNIQUE CHECK(POSITION('@' IN email)>1),
+    username VARCHAR(255) NOT NULL,
+    firstname VARCHAR(255) NOT NULL,
+    lastname VARCHAR(255) NOT NULL,
+    password VARCHAR(255) NOT NULL,
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE TABLE sleep (
+    id SERIAL PRIMARY KEY,
+    start_time TIMESTAMP NOT NULL DEFAULT NOW(),
+    end_time TIMESTAMP NOT NULL DEFAULT NOW(),
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    user_id INTEGER NOT NULL,
+    FOREIGN KEY (user_id) REFERENCES users(id)
```

Comment:
replace these two lines with:
`user_id    INTEGER REFERENCES users(id) ON DELETE CASCADE`
(on delete cascade will remove entries in this table if the primary key record (a user) is removed from the database)